sdk/tests/integration/HttpClientIntegrationTests.cc

```diff
@@ -0,0 +1,80 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "BaseIntegrationTest.h"
+#include "impl/HttpClient.h"
+
+#include <chrono>
```

Comment:
Is anything in `<chrono>` being used in this file?

Reply:
I intended to use timing for the REST request but can be removed! Thanks!